covidtask/covidtask/serializers.py

```diff
@@ -0,0 +1,26 @@
+from rest_framework import serializers
+from .models import Country
+from django.contrib.auth.models import User
+
+class CountrySerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Country
+        fields = "__all__"
+
+class SubscribtionSerializer(serializers.Serializer):
```

Comment:
let allow user to subscribe to one country at a time. This will require updating the serializer to a ModelSerializer, and the API View to a CreateAPIView and will become much cleaner.